strata/mongo/lreplica/replica.go

```diff
@@ -22,37 +22,45 @@ import (
 )
 
 type sessionGetter interface {
-	get(port string) (*mgo.Session, error)
+	get(string, string, string) (*mgo.Session, error)
```

Comment:
nit: since it's an interface, it could use some documentation on the three parameters :)